services/src/openkilda-gui/README.md

```diff
@@ -11,13 +11,13 @@ This project holds the OPEN KILDA GUI service for Open Kilda Controller.
 #### Configuration
 
 * The **_openkilda-gui_** feature must be installed in OPEN KILDA.
-* The GUI listens on port 1010
```

Comment:
your service still use port 1010, if this document states how to configure openkilda-gui, it must refer port 1010. If this document refer how to use local devel environment you should describe steps how to run it using docker-compose and/or make commands.

Reply:
Yep, I already defined server port is 1010 but when we build using docker-compose or make command then port 8010 listens.